### ISSUE TITLE
Add faq section at footer

### ISSUE THREAD

**Issue by ananyag309** [VesperAkshay/qr-code-generator#111]
The task involves incorporating a Frequently Asked Questions (FAQ) section in the footer of a website. This requires identifying key questions users might have, organizing the content clearly, and ensuring it's easily accessible from all pages.

**Comment by VesperAkshay** [comment]
@ananyag309 Assigned the issue

**Comment by Akanshaku** [comment]
I would like to work on this issue @VesperAkshay

**Comment by VesperAkshay** [comment]
Its already assigend @Akanshaku

**Comment by bj-jiwrajka** [comment]
> Its already assigend @Akanshaku

Hey @ananyag309, are you working on it?

**Comment by ananyag309** [comment]
Yes I am, sorry for taking time I was a bit occupied by my sem exam but I will raise the pr asap

**Comment by bj-jiwrajka** [comment]
> Yes I am, sorry for taking time I was a bit occupied by my sem exam but I will raise the pr asap

cool 👌all the best

**Comment by ananyag309** [comment]
Thank you so much 😊
